lib/msf/java/rmi/builder.rb

```diff
@@ -0,0 +1,96 @@
+# -*- coding: binary -*-
+
+require 'rex/java/serialization'
+
+module Msf
+  module Java
+    module Rmi
+      module Builder
+        # Builds a RMI header stream
+        #
+        # @param opts [Hash{Symbol => <String, Fixnum>}]
```

Comment:
There isn't ```Rex::Proto::Rmi::Model::OutputHeader#initialize``` but ```Rex::Proto::Rmi::Model::Element#initialize```. Because of it I prefer to add the Documentation for the hash of options here.